packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -893,6 +900,29 @@ Or read our docs at https://www.prisma.io/docs/concepts/components/prisma-client
       })
     }
 
+    _applyResultExtensions(result: object | null, modelName: string, args: JsArgs) {
```

Comment:
+1 for moving it to another file near applyQueryExtensions if possible.

Reply:
Done.